src/evict/evict_lru.c

```diff
@@ -2197,7 +2197,7 @@ __wt_cache_eviction_worker(WT_SESSION_IMPL *session, bool busy, u_int pct_full)
 	if (timer)
 		__wt_epoch(session, &enter);
 
-	for (init_evict_count = cache->pages_evict;; ret = 0) {
+	for (init_evict_count = cache->pages_evict, yield_count = 0;; ret = 0) {
```

Comment:
I'm with @sulabhM - since the `yield_count` isn't incremented as part of the loop, I think initializing it around line 2231 makes more sense.